routes/web.php

```diff
@@ -32,5 +32,7 @@
 Route::prefix('app')->middleware(['auth:sanctum', 'verified'])->name('app.')->group(function () {
     Route::get('team', [TeamController::class, 'index'])->name('team.index');
     Route::get('team/create', [TeamController::class, 'create'])->name('team.create');
+    Route::post('team/join', [TeamController::class, 'join'])->name('team.join');
+    Route::get('team/join-form', [TeamController::class, 'joinForm'])->name('team.joinForm');
```

Comment:
I would put both of them on the URL (`team/join`).

I would also recommend putting both methods in a a new Controller `TeamJoinController` for instance, with `create` and `store` methods: (`php artisan make:controller TeamJoinController`).

```suggestion
    Route::get('team/join', [TeamJoinController::class, 'create'])->name('team.join');
    Route::post('team/join', [TeamJoinController::class, 'store']);
```